test/testinfra.py

```diff
@@ -37,10 +37,10 @@
 from testimagetask import GithubImageTask
 
 TOKEN = "~/.config/github-token"
-topdir = os.path.normpath(os.path.dirname(os.path.realpath(__file__)))
+TEST_DIR = os.path.normpath(os.path.dirname(os.path.realpath(os.path.join(__file__, ".."))))
```

Comment:
Having the constant is nice, but in this case I think `os.path.join(__file__, "..")` is premature - the test dir is still the same as `__file__` is in.
I'm fine that this pull request introduces the constant, but changing the target should happen together with the refactoring.